syntax = "proto3";

option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "ModifyOrderResponseProto";
message ModifyOrderResponse {
	string orderId = 1; //订单编号
}